content/en/real_user_monitoring/rum_explorer.md

```diff
@@ -0,0 +1,145 @@
+
+##### Numerical values
+
+Use `<`,`>`, `<=`, or `>=` to perform a search on numerical attributes. For instance, retrieve all views that have a duration over 100ms with:
+
+`@duration:>100`
+
+You can search for numerical attribute within a specific range. For instance, retrieve all your views with a duration between 100ms and 300ms:
+
+`@duration:[100 TO 300]`
+
+##### Autocomplete
+
+Use the search bar's autocomplete feature to complete your query using existing values:
+
+{{< img src="real_user_monitoring/rum_explorer/search_bar_autocomplete.png" alt="search bar autocomplete " responsive="true" style="width:60%;">}}
+
+#### Examples
+
+| Search query                                                 | Description                                                                                                                                         |
+|--------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| `@http.url_details.path:"/api/v1/test"`                      | Searches all views containing `/api/v1/test` in the attribute `http.url_details.path`.                                                              |
+| `@http.url:\/api\/v1\/*`                                     | Searches all views containing a value in `http.url` attribute that start with `/api/v1/`                                                            |
+| `@duration:[100 TO 300] @http.url_details.path:\/api\/v1\/*` | Searches all views with a `duration` between 100 and 300 ms, and containing a value in `http.url_details.path` attribute that start with `/api/v1/` |
+
+## Vizualisation
+
+Click on any view to open the views panel and see more details about it: Resources, Traces, Errors, User Action, Long task, Logs, and Attributes:
+
+{{< img src="real_user_monitoring/rum_explorer/rum_views.png" alt="Rum View" responsive="true" style="width:80%;">}}
+
+## Facets & Measures
+
+After [being collected][1], your views attributes can be indexed as facets or measures in order to be accessible for your [context](#context) creation and [Analytics][2].
+
+Note: To leverage the most out of your Views explorer page, make sure your views attributes follow [Datadog attribute naming convention][3].
+
+{{< tabs >}}
+{{% tab "Facets" %}}
+
+A facet displays all the distinct members of an attribute or a tag and provides some basic analytics, such as the number of views represented. Facets allow you to pivot or filter your datasets based on a given attribute. To filter, select the values that you want to see.
+
+{{< img src="real_user_monitoring/rum_explorer/rum_facet.png" alt="Facets demo" responsive="true" style="width:80%;">}}
+
+**Create a Facet**:
+
+To start using an attribute as a facet or in the search, click on it and add it as a facet in the:
```

Comment:
Not sure what belongs after the, but flagging as it seems something is missing here.